src/components/ButtonWithMenu.js

```diff
@@ -24,9 +24,16 @@ const propTypes = {
     isDisabled: PropTypes.bool,
 
     /** Menu options to display */
-    /** [{text: 'Pay with Expensify', icon: Wallet}, {text: 'PayPal', icon: PayPal}, {text: 'Venmo', icon: Venmo}] */
+    /** [
+     *   {text: 'I\'ll settle up elsewhere', paymentType: 'Elsewhere', icon: Cash},
+     *   {text: 'Pay with Expensify', paymentType: 'Expensify', icon: Wallet},
+     *   {text: 'PayPal', paymentType: 'PayPal.me', icon: PayPal},
+     *   {text: 'Venmo', paymentType: 'Venmo', icon: Venmo}
+     *  ]
+     * */
```

Comment:
@nickmurray47 What I have ended up doing is adding a `paymentType` key to the options, which matches one of [these](https://github.com/Expensify/Web-Expensify/blob/bbd7178e9c2c03209b03ffebfbb69c4b5a293313/lib/Report/Action/IOU.php#L10-L13), which we originally missed. We have passed the `iouType` instead of  `paymentType` (the names are confusingly similar)